bin/create-docs-pr.js

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { program } = require('commander')
+
+const Github = require('./github')
+const git = require('./git-commands')
+
+const DEFAULT_FILE_NAME = 'NEWS.md'
+/** e.g. v7.2.1 */
+const TAG_VALID_REGEX = /v\d+\.\d+\.\d+/
+
+program.requiredOption('--tag <tag>', 'tag name to create GitHub release for')
+program.option('--remote <remote>', 'remote to push branch to', 'origin')
+program.option('--username <github username>', 'Owner of the fork of docs-website')
+program.option(
+  '--changelog <changelog>',
+  'Name of changelog(defaults to NEWS.md)',
+  DEFAULT_FILE_NAME
+)
+program.option('-f --force', 'bypass validation')
+program.option('--dry-run', 'executes script but does not commit nor create PR')
+program.option(
+  '--repo-path <path',
+  'Path to the docs-website fork on local machine',
+  'docs-website'
+)
+const RELEASE_NOTES_PATH =
+  './src/content/docs/release-notes/agent-release-notes/nodejs-release-notes'
+
+async function createRelease() {
+  // Parse commandline options inputs
+  program.parse()
+
+  const options = program.opts()
+
+  console.log(`Script running with following options: ${JSON.stringify(options)}`)
+
+  try {
+    const version = options.tag.replace('refs/tags/', '')
+    console.log(`Getting version from tag: ${version}`)
+
+    logStep('Validation')
+    validateTag(version, options.force)
+    logStep('Get Release Notes from File')
+    const { body, releaseDate } = await getReleaseNotes(version, options.changelog)
+    logStep('Branch Creation')
+    const branchName = await createBranch(options.repoPath, version, options.dryRun)
+    logStep('Format release notes file')
+    const releaseNotesBody = formatReleaseNotes(releaseDate, version, body)
+    logStep('Create Release Notes')
+    await addReleaseNotesFile(releaseNotesBody, version)
```

Comment:
Hard to tell just reading the code but seems like in this case the version would be `Major.minor.patch`. For the file naming convention, we want `Major-minor-patch` which doesn't seem like that conversion happens anywhere. But I may just not be interpreting things right

Reply:
oh, good catch. yes currently it is `major.minor.patch` ill change to be `major-minor-patch`